### PR TITLE
Added Trivia widget.

### PR DESCRIPTION
**Pull request by runishkumar**

### CODE REVIEW

File: src/dataSources/dataSources.js [hunk 1]
```diff
@@ -23,6 +23,14 @@ const dataSources = [
     fetchFunction: () => fetchUrl('http://www.mocky.io/v2/596f52eb0f00008d036b7535'),
   },
   { key: 'customFunnel', fetchFunction: () => fetchUrl('http://www.mocky.io/v2/596f52eb0f00008d036b7535') },
+  {
+    key: 'trivia',
+    fetchFunction: () =>
+      fetchUrl('http://api.icndb.com/jokes/random').then(response => {
+        console.log(response);
```
Comment (rocketeric) [inline src/dataSources/dataSources.js]: Can remove this line.

**Comment by rocketeric** [comment]
Nice start, but please change to trivia for today's date at http://numbersapi.com/

File: server/src/service/triviaService.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import _get from 'lodash/get';
```
Comment (rocketeric) [inline server/src/service/triviaService.js]: Getting warning about unused var on this one. Please double check and remove.

**Comment by rocketeric** [comment]
@gkatsanos, I see your point. However:
The server and the client are two separate applications and should ideally be in two separate repos. Therefore it doesn't matter if the server uses a different lib as long as we do not mix within the same application. 
We decided to go with axios since it seems to be hailed as an improvement over whatwg-fetch, although the difference is of course rather small.

**Comment by gkatsanos** [comment]
Sure but it could create confusion to potential developers - having to use two different libraries for the same thing :) Axios is definitely nice and has several fancy features but I guess in this simple usecase whatwg-fetch could work just as well. (plus it's future-proof). Last but not least Axios is rather big library so adding it increases the codebase/complexity/deployment time.

**Pull request merged**